lib/vdsm/tool/configurators/multipath.py

```diff
@@ -372,15 +370,13 @@ def isconfigured():
     """
     Check the multipath daemon configuration at /etc/multipath.conf.
     """
-
-    if os.path.exists(_CONF_FILE):
+    if os.path.exists(_CONF_FILE) and os.stat(_CONF_FILE).st_size > 0:
         with open(_CONF_FILE) as f:
-            mpathconf = [x.strip("\n") for x in f.readlines()]
-        try:
-            conf_check = _check_conf_rev(mpathconf)
-            return YES if conf_check else NO
-        except IndexError:
-            pass
+            first, second = [x.strip("\n")
+                             for x in (f.readline() for _ in range(2))]
+
+        conf_check = _check_conf_rev(first, second)
+        return YES if conf_check else NO
```

Comment:
This code should be kept as is, just remove the error handler since _check_conf_revision() should handle it.

    if os.path.exists(_CONF_FILE):
        with open(_CONF_FILE) as f:
            mpathconf = [x.strip("\n") for x in f.readlines()]
            revision_ok = _check_revision(first, second)
            ...